### PR TITLE
Lin reg exer

### PR DESCRIPTION
**Pull request by canyon289**
Add Linear regression exercise set,

This is the only file that matters [Linear_Regression/Linear_Regression.ipynb](https://github.com/AlexAndorra/notebooks-experiments/pull/9/files#diff-4ea0a0bf6d84d12f8bee1c97a60fe1dbba4349f4e94e81741ae61f0255f5e5fc)

### CODE REVIEW

**Comment by AlexAndorra** [comment]
Thanks @canyon289 , I love it! The dataset is fun. Where did you find it?
Left a few comments below, but already looking great 🤩

File: Linear_Regression/LinearRegression Exercises.ipynb [hunk 1]
```diff
@@ -0,0 +1,1442 @@
+{
```
Comment (AlexAndorra) [inline Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>That would actually be good to "trap" them and not remove the missing data. They'll see what a failing model looks like and how to fix it ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (AlexAndorra) [inline Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Just quickly say what these two variables are</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (AlexAndorra) [inline Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Copy pasting artefact: "# each species get a slope for each predictor thx to <code>dims</code>"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (AlexAndorra) [inline Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>This is the same model as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='20'/>

Comment (AlexAndorra) [inline Linear_Regression/LinearRegression Exercises.ipynb:1]: <ul><li>Same copy pasting artefact as previously</li><li><code>as adelie_regression</code> --&gt; not anymore ;) </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='24'/>

Comment (canyon289) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Added above, I love the idea but the model is not failing unfortunately, PyMC imputes the values and the students might miss the point</p>

Comment (canyon289) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Dont I say it above?</p><p> </p>

Comment (canyon289) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Thanks fixed</p><p> </p>

**Comment by canyon289** [comment]
@AlexAndorra thanks Alex. Found it as part of the book research, it is quite a nice dataset

File: Linear_Regression/LinearRegression Exercises.ipynb (hunk 1 above)
Comment (AlexAndorra) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Really? So there are NaNs for some observations and PyMC imputes them without any masked array stuff??</p><p> </p>

Comment (AlexAndorra) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: <p>Mainly: what does depth represent for a penguin? Length is self explanatory</p>

Comment (canyon289) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: Yep it seems like it, Ill just let them do the filtering themselves per your EDA comment

Comment (canyon289) [reply Linear_Regression/LinearRegression Exercises.ipynb:1]: Got it will update

**Pull request closed without merging**